modules/detect_target/detect_target_brightspot.py

```diff
@@ -0,0 +1,110 @@
+"""
+Detects bright spots generated by an IR camera.
+"""
+
+import time
+
+import cv2
+import numpy as np
+
+from . import base_detect_target
+from .. import image_and_time
+from .. import detections_and_time
+
+
+BRIGHTSPOT_THRESHOLD = 240
```

Comment:
Could you make this a parameter that's passed in from config.yaml? It may be easier to test that way. For your unit tests you can continue to use a hard coded value. cc @Xierumeng do we want to modify detect target factory to have a dict (kind of like kwargs?) for these kinds of things?

Reply:
I dislike kwargs because it removes information from a function signature. Perhaps a data exchange format for the configs? For example:
`detect_target_ultralytics.py`
```py
class DetectTargetUltralyticsConfig:
    def __init__(self, thing, other_thing):
        ...

class DetectTargetUltralytics:
    @classmethod
    def create(cls, common_thing, another_common, config: DetectTargetUltralyticsConfig):
        ...
```
`detect_target_brightspot.py`
```py
class DetectTargetBrightspotConfig:
    def __init__(self, other_thing, setting):
        ...

class DetectTargetBrightspot:
    @classmethod
    def create(cls, common_thing, another_common, config: DetectTargetBrightspotConfig):
        ...
```
`detect_target_factory.py`
```py
def create_detect_target(detect_target_option: DetectTargetOption, common_thing, another_common, config: DetectTargetUltralyticsConfig | DetectTargetBrightspotConfig):
    match detect_target_option:
        case DetectTargetOption.ML_ULTRALYTICS:
            return detect_target_ultralytics.DetectTargetUltralytics.create(common_thing, another_common, config)
        case DetectTargetOption.C_BRIGHTSPOT:
            return detect_target_brightspot.DetectTargetBrightspot.create(common_thing, another_common, config)

    return False, None
```